ParisBNF/abb/BNFabb91.xml

```diff
@@ -0,0 +1,432 @@
+                       <locus from="87r" to="102v"/>
+                       <seg type="script">15th century.</seg>
+                       <seg type="ink">Black, red.</seg>
+                       <desc>Careful, somewhat irregular; by a mediocre scribe.                        
+                          Similar, but probably not identical to <ref target="#h1"/>. The numerals are written with dashes above and below 
+                          (cp. <locus target="#92ra #94ra #97ra"/>).</desc>
+                    </handNote>
+                 </handDesc>
+                  <decoDesc>
+                     
+                     <decoNote type="ornamentation" xml:id="d1">
+                        <locus target="#1r"/>
+                        <desc>Ornamental band made of dots and lines, possibly incomplete.</desc>
+                     </decoNote>
+                     
+                     
+                     <decoNote type="ornamentation" xml:id="d2">
+                        <locus target="#28vb"/>
+                        <desc>Narrow ornamental band.</desc>
+                     </decoNote>
+                     
+                     
+                     <decoNote type="ornamentation" xml:id="d3">
+                        <locus target="#37r"/>
+                        <desc>Fine ornamental band.</desc>
+                     </decoNote>
+                     
+                     <decoNote type="drawing" xml:id="d4">
+                        <locus target="#93v"/>
+                        <desc>A doodle or a small drawing, possibly protective.</desc>
+                     </decoNote>
+                     
+                     <decoNote type="drawing" xml:id="d5">
+                        <locus target="#103r #104v"/>
+                        <desc>Doodle, trials of drawing ornamental bands, small drawing of an animal.</desc>
+                     </decoNote>
+                     
+                  </decoDesc>
+                     
+
+
+
+                 <additions>
+                   <list>                     
+                                            
+                      <item xml:id="a1">
+                         <locus target="#9r"></locus>
+                         <desc type="Unclear">One or possibly two notes in the bottom margin, 
+                            written in a crude hand, half-erased and poorly readable.</desc>
+                         <q xml:lang="gez"/>
+                      </item>
+                      
+                      <item xml:id="a2">
+                         <locus from="86va" to="86vb"/>
+                         <desc type="Unclear">A note written in an old crude hand mentioning, of the content which is not fully clear, mentioning ʾabuna Takla Ṣǝyon, probably a 
+                            senior ecclesiastic. Despite  mentioning<foreign xml:lang="gez">መጽሐፈ፡ ተጋንዮ፡</foreign>, the relation of the note
+                            to the manuscript is not clear.</desc>
```

Comment:
the second sentence seems to somewhat repeat the first? delete or modify the first "of the content which is not fully clear"?